velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -421,20 +424,39 @@ std::optional<TimestampUnit> getTimestampUnit(
   return std::nullopt;
 }
 
+std::optional<std::string> getTimestampTimeZone(
+    const Config& config,
+    const char* configKey) {
+  if (const auto timezone = config.get<std::string>(configKey)) {
+    return std::optional(static_cast<std::string>(timezone.value()));
```

Comment:
why unpack then repack the optional? Could you just return it?

```
  return timezone;
```

Reply:
actually, seems like the whole function could be:

```
return config.get<std::string>(configKey);
```

Reply:
@pedroerp `config.get<std::string>(configKey)` return` folly::Optional<T>` not `std::optional<std::string>`. And we also need to check whether timezone has value.